components/popupSearch/index.tsx

```diff
@@ -66,21 +66,18 @@ export const PopupSearch: React.FC<HeaderProps> = ({ isPopupOpen, setIsPopupOpen
               placeholder="You can search for ‘Kyoukai no Kanata’ for example"
             />
             <SearchList>
-              {searchList && (
-                <>
-                  {searchList?.map(({ images, title, mal_id, episodes, status, type }) => (
-                    <SearchItem
-                      key={mal_id}
-                      id={mal_id}
-                      type={type}
-                      status={status}
-                      image={images?.jpg?.small_image_url}
-                      title={title}
-                      episodes={episodes}
-                    />
-                  ))}
-                </>
-              )}
+              {searchList &&
+                searchList?.map(({ images, title, mal_id, episodes, status, type }) => (
```

Comment:
do you still need 
searchList &&
  searchList?.map
both conditions?
I think you can remove "searchList &&" check and it will work the same

Especially as I mentioned it in comment: https://github.com/emfree1/Anime-Next-site/pull/1#discussion_r976544768 it will always be true as it is Array https://github.com/emfree1/Anime-Next-site/blob/80fa2d8dc2be68a1ae1ab8296c40ef8c50814564/components/popupSearch/index.tsx#L19

Why this was not changed?

Reply:
found in another commit